tests/test_job.py

```diff
@@ -123,6 +123,35 @@ def test_list(self, connection, add_response, limit, ids, want_params, want_name
         have_params = responses.calls[0].request.params  # pyright: reportGeneralTypeIssues=false
         assert have_params == want_params
 
+    @pytest.mark.parametrize(
+        "limit, status, want_status_param",
+        [
+            (1, "queued", "queued"),
+            (
+                2,
+                None,
+                None,
+            ),
+            (2, "invalid", "invalid"),
+            (
+                2,
+                "complete",
+                "complete",
+            ),
```

Comment:
Let's be consistent about the trailing commas:
```suggestion
            (1, "queued", "queued"),
            (2, None, None),
            (2, "invalid", "invalid"),
            (2, "complete", "complete"),
```